packages/components/src/button/button.styles.ts

```diff
@@ -661,7 +660,6 @@ export const buttonStyles: (
     appearanceBehavior(
       'stealth',
       css`
-        :host([appearance='stealth'][disabled]),
```

Comment:
We should remove the all block including on hover and on active case as the goal is to avoid style change when the button is disabled.